### DISCUSSION TITLE
How do I abbreviate Fig. and Eqn. in Quarto crossreferences?

### DISCUSSION THREAD

**Discussion by jessicaguo** [quarto-dev/quarto-cli#7000] [Q&A] [answered]
### Description

The journals in my area of study haven't published quarto templates yet. In the meantime, how can I get the cross-references to abbreviate to Fig. and Eqn.?

**Comment by mcanouil** [comment]
Use:

````qmd
---
language:
  crossref-fig-title: Fig
---
````

See <https://quarto.org/docs/authoring/language.html#alternate-language>.

EDIT: directly using `crossref` key as mentioned by @cderv (https://github.com/quarto-dev/quarto-cli/discussions/7000#discussioncomment-7112283)

````qmd
---
crossref:
  fig-title: Fig
---
````

See <https://quarto.org/docs/authoring/cross-references.html#options>.

**Comment by cderv** [comment, marked as answer]
Using `language` is probably working too, but I believe the right way to do that is tweaking the `crossref` option directly 
https://quarto.org/docs/authoring/cross-references.html#options

Especially with new crossref system in next 1.4 version, the `crossref` key should be the one to target
https://quarto.org/docs/prerelease/1.4/crossref.html#custom-crossreferenceable-types

**Reply by mcanouil** [reply]
By the way, is possible to localised a custom cross-ref? (cc @cscheid)

**Reply by cscheid** [reply]
> By the way, is possible to localised a custom cross-ref? (cc @cscheid)

Good question. You are supposed to supply "Video", "Diagram", etc, and so you can provide the localized versions of those directly. But there currently isn't a direct way to do it in multi-language setups. We can leverage project profile metadata for it, but it might be a good idea to do this correctly. Let's discuss this in a meeting.
